src/react-growsumojs.js

```diff
@@ -1,4 +1,4 @@
-export const initialize = key => {
+export const initialize = (key,cb) => {
```

Comment:
We must add cb, we don't know when the script was loaded, so the call to  GrowSumo.createSignup will fail in some case, depend on the timing
if you call GrowSumo.createSignup on cb function you will know 100% the script was loaded